connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -469,15 +470,22 @@ function formatDocumentContent({
 }): CoreAPIDataSourceDocumentSection {
   const URL_MAX_LENGTH = 128;
   const TITLE_MAX_LENGTH = 300;
-  const parsedUrl = new URL(url);
+
+  const validatedUrl = validateUrl(url);
+  if (!validatedUrl.valid || !validatedUrl.standardized) {
+    throw new Error(`Invalid URL: ${url}`);
```

Comment:
Will this stop the workflow? We should just ignore here right?

Reply:
Reading the code it seems it will not stop the workflow

Reply:
Indeed, will skip in this case